app/models/Submission.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+import java.time.LocalDateTime
+
+import play.api.libs.json.Json
+
+case class Submission(
+  conversationId: String,
+  ucr: String,
+  submissionType: String,
+  submissionAction: String,
+  dateUpdated: LocalDateTime,
```

Comment:
Similar to above, the recommended approach with Dates is to use `Instant`

Reply:
`LocalDateTime` vs `ZonedDateTime`
I am getting confused now. Please confirm :)

Reply:
ZonedDateTime is better than LocalDateTime, but Instant makes everyones life easier

Reply:
For the avoidance of any doubt I suggest we *always* use `Instant` :)